docs/blog/posts/2024/srlinux-asymmetric-routing.md

```diff
@@ -0,0 +1,1801 @@
+                address 198.51.100.5/31 {
+                }
+            }
+        }
+    }
+    interface ethernet-1/3 {
+        admin-state enable
+        mtu 9100
+        subinterface 0 {
+            admin-state enable
+            ipv4 {
+                admin-state enable
+                address 198.51.100.9/31 {
+                }
+            }
+        }
+    }
+    interface ethernet-1/4 {
+        admin-state enable
+        mtu 9100
+        subinterface 0 {
+            admin-state enable
+            ipv4 {
+                admin-state enable
+                address 198.51.100.13/31 {
+                }
+            }
+        }
+    }
+```
+
+///
+
+/// tab | spine2
+
+```{.srl .code-scroll-lg}
+A:spine2# info interface ethernet-1/{1..4}
+    interface ethernet-1/1 {
+        admin-state enable
+        mtu 9100
+        subinterface 0 {
+            admin-state enable
+            ipv4 {
+                admin-state enable
+                address 198.51.100.3/31 {
+                }
+            }
+        }
+    }
+    interface ethernet-1/2 {
+        admin-state enable
+        mtu 9100
+        subinterface 0 {
+            admin-state enable
+            ipv4 {
+                admin-state enable
+                address 198.51.100.7/31 {
+                }
+            }
+        }
+    }
+    interface ethernet-1/3 {
+        admin-state enable
+        mtu 9100
+        subinterface 0 {
+            admin-state enable
+            ipv4 {
+                admin-state enable
+                address 198.51.100.11/31 {
+                }
+            }
+        }
+    }
+    interface ethernet-1/4 {
+        admin-state enable
+        mtu 9100
+        subinterface 0 {
+            admin-state enable
+            ipv4 {
+                admin-state enable
+                address 198.51.100.15/31 {
+                }
+            }
+        }
+    }
+```
+
+///
+
+/// admonition | CLI Ranges
+    type: tip
+Notice that configuration for multiple interfaces are shown with a single command using the concept of ranges.  
+Different ways of using ranges are shown with one style used for the leafs and another for the spines. With `interface ethernet-1{1,2}`, the comma-separation allows the user to enter any set of numbers (contiguous or not), which are subsequently expanded. Thus, this expands to `interface ethernet-1/1` and `interface ethernet-1/2`.
+
+On the other hand, you can also provide a contiguous range of numbers by using `..`, as shown for the spines. In that case, `interface ethernet-1/{1..4}` implies ethernet-1/1 through ethernet-1/4.
+
+Check out a separate post on [CLI Ranges and Wildcards](../2023/cli-ranges.md).
+///
+
+Remember, by default, there is no global routing instance/table in SR Linux. A `network-instance` of type `default` must be configured and these interfaces, including the `system0` interface need to be added to this network instance for point-to-point connectivity.
```

Comment:
from Jorge

about this: "Remember, by default, there is no global routing instance/table in SR Linux" -> this is confusing since the default network instance is really equivalent to the global routing table.

Reply:
Agreed, but there is no default network instance defined as of 24.7.1. Only the mgmt network-instance exists by default, which is why I made a note of ensuring the user creates this default network-instance.

Reply:
I'd change it to 

_A `network-instance` named `default` must be configured..._

since the default type will be implicitly set for a netinst named `default`